app/controllers/application_controller.rb

```diff
@@ -31,4 +31,11 @@ def can_edit_event(event_id)
     event = Event.find(event_id)
     can_edit_event_type(event.event_type.id)
   end
+
+  def can_create_events
+    if current_user.position == nil
+      flash[:alert] = 'You must be a chairman or the secretary to add an event'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.